e2e/tests/api-driven/src/globalHelpers.ts

```diff
@@ -0,0 +1,57 @@
+import { TEST_EMAIL } from "../../ui-driven/src/helpers";
+import { $admin } from "./client";
+
+export function createTeam(
+  args?: Partial<Parameters<typeof $admin.team.create>[0]>,
+) {
+  return safely(() =>
+    $admin.team.create({
+      name: "E2E Test Team",
+      slug: "E2E",
+      logo: "https://raw.githubusercontent.com/theopensystemslab/planx-team-logos/main/planx-testing.svg",
+      primaryColor: "#444444",
+      submissionEmail: TEST_EMAIL,
+      homepage: "planx.uk",
+      ...args,
+    }),
+  );
+}
+
+export function createUser(
+  args?: Partial<Parameters<typeof $admin.user.create>[0]>,
+) {
+  return safely(() =>
+    $admin.user.create({
+      firstName: "Test",
+      lastName: "Test",
+      email: TEST_EMAIL,
+      ...args,
+    }),
+  );
+}
+
+export function createFlow(
+  args: Omit<Parameters<typeof $admin.flow.create>[0], "data">,
+) {
+  return safely(() =>
+    $admin.flow.create({
+      data: { dummy: "flowData " },
+      ...args,
+    }),
+  );
+}
+
+/**
+ * Error handling boilerplate for client functions
+ */
+export function safely<T extends () => ReturnType<T>>(callback: T) {
```

Comment:
One thing this highlighted was that the `planx-core` client doesn't always return consistent values for success/failure. Sometimes it's a `null`, or `false`, or an error is thrown.

Taking a bit of inspiration from `useSWR` it would be super nice if it just always returned a consistent result object of some sort which we could check for errors.